### PR TITLE
FY2020Q1 quarterly refresh

### PR DESCRIPTION
**Pull request by andrewconnell**
- Updated all packages versions
- Updated exercise instructions
- Updated slide decks

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,6 +7,4 @@ obj/
 !./.vscode/settings.json
 !./.vscode/tasks.json
 !./.vscode/launch.json
-!./.vscode/extensions.json
-
-appsettings.json
\ No newline at end of file
+!./.vscode/extensions.json
```
Comment (jasonjoh) [inline .gitignore]: appsettings.json should not be commited (has app ID and secret)

Comment (jasonjoh) [reply .gitignore]: You'll want to purge the appsettings.json files from the repo since they've been commited.

Comment (andrewconnell) [reply .gitignore]: Removed **appsettings.json** from all demos & added entry to **.gitignore** to ignore in the future. We left placeholders in there to address this, but removed regardless.

**Comment by andrewconnell** [comment]
@jasonjoh Just added the example files to the demos with updated README instructions. I think this is what you were requesting. If not, can you let me know and I'll fix?

**Pull request merged**